src/main/java/net/dv8tion/jda/api/entities/User.java

```diff
@@ -28,6 +28,7 @@
 import javax.annotation.CheckReturnValue;
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
+import java.awt.*;
```

Comment:
Reduce this Import to just Color, probably don't need to load the whole package.

Reply:
IntelliJ likes to just import the whole package >_> fixed